content/lore/elm-core-development.md

```diff
@@ -0,0 +1,186 @@
+
+
+### I guess the Elm community just wants to ignore/avoid this problem and not have `<good outcome>`.
+
+While we understand the frustration, ultimately this is a problem with accepting that:
+
+- We cannot control people
+- Other people make different choices than we might
+- `<good arguments>` and `<good outcomes>` don’t invalidate the prior two points
+
+This creates a fork in the road based on acceptance.
+
+Arguments and debates usually spring up between people who’ve chosen different paths:
+
+- Some people cannot accept Evan’s chosen approach, or the fact it means `<some action>` can’t/won’t happen right now/ever
+  - Some of those people have stopped using Elm and/or left the Elm community as a result
+  - Some of these departures have been very vocal and critical
+  - Some of that criticism has been picked up by people who were never in the community as a reason to never join, or discourage others from doing so
+  - This leads to repeated negative dialogue by some people anytime Elm is mentioned in certain forums: Reddit & HackerNews specifically repeats this pattern
+  - Some people in this group tend to re-emerge whenever this topic gets brought up and pile on inaccurate characterisations, derailing discussions
+- Some people choose to focus on the large area outside of core where they can contribute.
+  - **This does not imply everyone loves everything about Evan’s choices and hasn't been frustrated at some point**
+  - This is most of the active Elm community
+  - Many people choose to participate in building out a rich ecosystem on top of Elm core
+    - Some of these people are very excited about the future of Elm
+  - Some people choose explore new areas in a way that is compatible with Elm core
+    - [Lamdera](https://lamdera.com), a backwards/forwards-compatible soft-fork of the Elm compiler, with features for full-stack Elm and extensions for Elm tooling authors and Elm users
+    - [elm-pages](https://elm-pages.com) - JAMstack for elm.
+  - Some people take inspiration and start new initiatives not compatible with Elm core
+    - Gren, a hard fork of Elm by Robin Heggelund Hansen, previous contributor to various Elm projects
+
+
+The Elm community is not ignorant of the trade-offs of Evan’s choices – we just acknowledge they are the trade-offs he has chosen and communicated repeatedly.
+
+We’re also human: the tradeoffs may not be ideal, they may not be the ones we would have chosen, but there is no choice that pleases everyone: so we respect Evan’s right to choose.
+
+### Why doesn't someone talk to Evan and make him see sense?
+
+Arguments in this line seem to carry a few assumptions:
+
+- That Evan has never had these conversations before
+- That because Evan is obviously wrong, the only remaining issue is to badger him until he becomes rational and changes his mind
+
```

Comment:
This feels like its missing a conclusion.

```suggestion

Since the second has been tried a lot (unsuccessfully), we can safely conclude the first isn't the case.
```